src/store/user/actions.js

```diff
@@ -102,6 +102,18 @@ export const Signup = (name, email, password) => {
 
 export function updateScore(language, score){
   return async (dispatch, getState) => {
+
+    const token = selectToken(getState());
     dispatch({ type: 'UPDATE_SCORE', payload: { [language]: score } });
-  }
-}
+
+    try{
+      const response = await axios.patch('http://localhost:4000/scores', 
```

Comment:
you can import { apiUrl } from "../../config/constants"  and use `${apiUrl}/scores`. instead of localhost:4000